tensorrt_llm/_torch/visual_gen/cute_dsl_kernels/helpers/static_persistent_tile_scheduler.py

```
@@ -0,0 +1,814 @@
+            # Use get_flat_coord for swizzled layouts (FastDivmod doesn't support them)
+            cur_cluster_coord = self.params.problem_layout_ncluster_mnl.get_flat_coord(
+                current_work_linear_idx, loc=loc, ip=ip
+            )
+
+        cur_tile_coord = tuple(
+            cute.arch.make_warp_uniform(Int32(x) * Int32(z) + Int32(y))
+            for x, y, z in zip(
+                cur_cluster_coord,
+                self.cta_id_in_cluster,  # type: ignore[arg-type]
+                (*self.params.cluster_shape_mn, Int32(1)),
+            )
```

Ruff (0.15.20)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)